### ISSUE TITLE
Cannot specify use_weights=False in MAE for loss function and evaluation metric

### ISSUE THREAD

**Issue by tle4336** [catboost/catboost#2207]
Problem: Cannot specify use_weights=False in CatBoostRegressor for either custom_metric or loss_function for MAE function
catboost version: 0.26
Operating System: Windows 10 64x bit - Pycharm 2021.2 version
CPU: Yes
GPU: No

I followed exactly this tutorial for setting `use_weights=False` in MAE by trying both `metrics.MAE(use_weights=False)` and `customer_metric = [MAE: use_weights=False]`. None of them work for the `CatBoostRegressor()`. The error message I kept bumping into is either `If non-default weights for objects are not set, the 'use_weights' parameter must not be specified.` OR ` __call__() takes 1 positional argument but 2 were given`. The current parameters are just simply set like this:

`model = CatBoostRegressor(iterations=600, loss_function='MAE', custom_metric=metrics.MAE(use_weights=False), learning_rate=0.01, one_hot_max_size=2, l2_leaf_reg=0.003, random_seed=777)`

I am not sure why you guys provided some examples that cannot be replicated by users. Please help improve **significantly** your documentation to be exact so that people like us have a clue on how to follow.  
https://github.com/catboost/tutorials/blob/master/metrics/Metrics_tutorial.ipynb
